.gitignore

```diff
@@ -1,14 +0,0 @@
-__pycache__/
```

Comment:
I guess it was a mistake that the global `.gitignore` was deleted. Can you please revert.